site/src/_pages/spec/2_embedding-applications.mdx

```diff
@@ -1,5 +1,17 @@
 # Embedding applications
 
+<InfoCard>
+
+We have published [the source code of an example embedding application, HASH](https://github.com/hashintel/hash).
+
+This is very much a work in progress we are using to explore and test assumptions and suggested approaches in the Block Protocol.
+
+As we develop this application we explore more issues related to the application-block interface. We will provide abstractions for embedding applications to minimise the amount of work required to use blocks.
```

Comment:
```suggestion
As we develop this application we explore more issues related to the application-block interface. We will provide abstractions for embedding applications to minimize the amount of work required to use blocks.
```

Standardizing on American English 👀 Haven't done a full pass. Should at some point.